src/shared/services/configuration/endpoints.json

```diff
@@ -0,0 +1,22 @@
+{
+  "AREAS_ENDPOINT": "/signals/v1/private/areas/",
+  "AUTH_ME_ENDPOINT": "/signals/v1/private/me/",
+  "CATEGORIES_ENDPOINT": "/signals/v1/public/terms/categories/",
+  "CATEGORIES_PRIVATE_ENDPOINT": "/signals/v1/private/categories/",
+  "DEPARTMENTS_ENDPOINT": "/signals/v1/private/departments/",
+  "FEEDBACK_FORMS_ENDPOINT": "/signals/v1/public/feedback/forms/",
+  "FEEDBACK_STANDARD_ANSWERS_ENDPOINT": "/signals/v1/public/feedback/standard_answers/",
+  "FILTERS_ENDPOINT": "/signals/v1/private/me/filters/",
+  "GEOGRAPHY_ENDPOINT": "/signals/v1/private/signals/geography",
+  "INCIDENTS_ENDPOINT": "/signals/v1/private/signals/",
+  "INCIDENT_PRIVATE_ENDPOINT": "/signals/v1/private/signals/",
+  "INCIDENT_PUBLIC_ENDPOINT": "/signals/v1/public/signals/",
+  "PERMISSIONS_ENDPOINT": "/signals/v1/private/permissions/",
+  "PREDICTION_ENDPOINT": "/signals/category/prediction",
+  "QUESTIONS_ENDPOINT": "/signals/v1/public/questions/",
+  "ROLES_ENDPOINT": "/signals/v1/private/roles/",
+  "SEARCH_ENDPOINT": "/signals/v1/private/search",
+  "SOURCES_ENDPOINT": "/signals/v1/private/sources/",
+  "TERMS_ENDPOINT": "/signals/v1/private/terms/categories/",
+  "USERS_ENDPOINT": "/signals/v1/private/users/"
```

Comment:
All of the original descriptions in the schema are lost. Can they be added here as comments?

Reply:
Good one! How about this?